rskj-core/src/main/java/co/rsk/validators/TxsMinGasPriceRule.java

```diff
@@ -49,8 +50,9 @@ public boolean isValid(Block block) {
         if(CollectionUtils.isNotEmpty(block.getTransactionsList())) {
             for (Transaction tx : txs) {
                 if (!(tx instanceof RemascTransaction) && tx.getGasPriceAsInteger().compareTo(blockMgp) < 0) {
-                    logger.warn("Tx gas price is under the Min gas Price of the block");
-                    panicProcessor.panic("invalidmingasprice", "Tx gas price is under the Min gas Price of the block");
+                    String logMessage = String.format("Tx gas price is under the Min gas Price of the block tx=%s", Hex.toHexString(tx.getHash()));
```

Comment:
Same comment, why use String.format in one place, and log message with {} in other places?

Reply:
Here the idea was to avoid doing string formatting 2 times for warning and panic. Maybe in the other PR this panic disappears and makes less sense doing things like this.